test/utils.jl

```diff
@@ -28,6 +28,7 @@
         DX[2] = v
         @test DX[2] == v
         @test X[:, 2] == v
+        @test all(zero(DX) .== Ref(zeros(D)))
```

Comment:
IMO these are not good and appropriate tests here. We would like to ensure that
- the function returns a `ColVecs`/`RowVecs` object of the same type as the input
- the size of the returned object is as expected
- all elements are zeros

Reply:
Would
```suggestion
        @test all(iszero, zero(DX))
        @test zero(DX) isa ColVecs
        @test size(zero(DX)) == size(DX)
```
do @devmotion ?

Reply:
Yeah that would be better. Additionally, one could check the size of the elements. So maybe
```suggestion
        zero_DX = zero(DX)
        @test all(iszero, zero_DX)
        @test zero_DX isa ColVecs
        @test size(zero_DX.X) == size(DX.X)
```